src/api/routes/alerts.py

```
@@ -0,0 +1,49 @@
+async def get_alerts():
+    try:
+        return alert_manager.get_alerts()
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
+
+@router.get("/alerts/{alert_id}", response_model=Alert)
+async def get_alert(alert_id: int):
+    try:
+        return alert_manager.get_alert(alert_id)
+    except Exception as e:
+        raise HTTPException(status_code=404, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)